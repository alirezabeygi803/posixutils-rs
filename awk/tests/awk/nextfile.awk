$1 == 3 { nextfile }
{ print $0 }
END { print "end" }
