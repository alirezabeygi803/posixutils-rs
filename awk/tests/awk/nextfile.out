1 Jane janitor 30
2 Smith sailor 45
1 1 1
2 2 2
end
